docs/faq/contributing.txt

```diff
@@ -53,7 +53,7 @@ To determine the right time, you need to keep an eye on the schedule. If you
 post your message right before a release deadline, you're not likely to get the
 sort of attention you require.
 
-Gentle IRC reminders can also work -- again, strategically timed if possible.
+Gentle IRC/Discord reminders can also work -- again, strategically timed if possible.
 During a bug sprint would be a very good time, for example.
```

Comment:
This will need wrapping to comply with Django’s requirement to [wrap all docs at 79 characters](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#:~:text=Documentation%2C%20comments%2C%20and%20docstrings%20should%20be%20wrapped%20at%2079%20characters).

```suggestion
Gentle IRC/Discord reminders can also work -- again, strategically timed if
possible. During a bug sprint would be a very good time, for example.
```